src/messaging/ReliableMessageMgr.cpp

```diff
@@ -118,6 +118,10 @@ void ReliableMessageMgr::ExecuteActions()
 
     // Retransmit / cancel anything in the retrans table whose retrans timeout has expired
     mRetransTable.ForEachActiveObject([&](auto * entry) {
+        // Check if groups
+        if (entry->ec->IsGroupExchangeContext())
+            return Loop::Continue;
```

Comment:
Wait, how did a group context get into the table to start with?  It shouldn't be there, wasting memory....  If these lines are actually needed for some reason, and we can't easily fix the real bug (that the ec is in the table even though there is absolutely no point to it being there), we should at the very least have a TODO comment and tracking issue.

Reply:
I was able to find the issue when looking into again.